CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Bend 
+Thank you for considering contributing to Bend!
+
+## How to Contribute
+### Reporting bugs
+1. **Check for existing issues:** Before you create a new issue, please do a search in [our issues](https://github.com/HigherOrderCO/Bend/issues) to see if the issue or feature request has already been filed.
+2. **Create a new issue:** If you find no issue or your issue differs, [create a new issue](https://github.com/HigherOrderCO/Bend/issues/new?template=bug_report.yml) and provide detailed information, including steps to reproduce the problem.
+
+### Suggesting Enhancements
+1. **Check for existing suggestions:** Before suggesting a new feature, please check if it's already been suggested in [our issues](https://github.com/HigherOrderCO/Bend/issues).
+2. **Create a new suggestion:** If you find no similar suggestion, [create a new suggestion issue](https://github.com/HigherOrderCO/Bend/issues/new?template=feature_request.md) and provide detailed information about the enhancement and why it would be useful.
+
+TODO
```

Comment:
We should also add a section about sending changes, explaining that:
* Changes are done through github pull requests. You have to fork and send the commit to your own branch.
* Before running, you have to run `cargo fmt` and `cargo clippy` to format and lint your code and `cargo insta test` and `cargo insta review` to run the tests and save any changes to the test results.